.github/ISSUE_TEMPLATE/config.xml

```diff
@@ -0,0 +1,18 @@
+blank_issues_enabled: false
+contact_links:
+  - name: "Support tickets"
+    url: https://www.odoo.com/help
+    about: "Please favour opening an official ticket if you have an active subscription."
+  - name: "Code contribution"
+    url: https://www.odoo.com/documentation/master/contributing/development.html#make-your-first-contribution
+    about: "If you have an idea of the change to apply in the codebase, please open a pull request to fasten the process."
```

Comment:
fasten (`your seat belt` :D) doesn't seem the appropriate word.
`Speed up` might be more adequate :eyes:

(Mb, should have noticed before)